shell/components/form/LabeledSelect.vue

```diff
@@ -284,6 +285,11 @@ export default {
         <template v-else-if="option.kind === 'divider'">
           <hr />
         </template>
+        <template v-else-if="option.kind === 'link'">
```

Comment:
Can we use a different term from link, as this is just highlighted and not a link?